Gemfile

```diff
@@ -45,6 +45,11 @@ gem "savon"
 # Render smart quotes
 gem 'rubypants'
 
+# For the API
+gem "blueprinter"
+gem "oj"
+gem "pagy"
```

Comment:
I know this is just a spike but we don't need to specify `pagy` as it's already pulled in by `govuk-components`.